java/system-test/system-test-dsl/src/test/java/sleeper/systemtest/dsl/util/WaitForJobsStatusTest.java

```diff
@@ -85,35 +87,43 @@ void shouldReportSeveralCompactionJobs() {
         // Given
         CompactionJob createdJob = compactionJob("created-job", "1.parquet", "2.parquet");
         CompactionJob startedJob = compactionJob("started-job", "3.parquet", "4.parquet");
-        CompactionJob finishedJob = compactionJob("finished-job", "5.parquet", "6.parquet");
+        CompactionJob uncommittedJob = compactionJob("uncommitted-job", "5.parquet", "6.parquet");
+        CompactionJob finishedJob = compactionJob("finished-job", "7.parquet", "8.parquet");
         store.fixUpdateTime(Instant.parse("2023-09-18T14:47:00Z"));
         store.jobCreated(createdJob);
         store.jobCreated(startedJob);
+        store.jobCreated(uncommittedJob);
         store.jobCreated(finishedJob);
-        store.fixUpdateTime(Instant.parse("2023-09-18T14:48:02Z"));
-        store.jobStarted(compactionJobStarted(finishedJob, Instant.parse("2023-09-18T14:48:00Z")).taskId("finished-task").build());
-        store.jobStarted(compactionJobStarted(startedJob, Instant.parse("2023-09-18T14:48:01Z")).taskId("started-task").build());
-        store.fixUpdateTime(Instant.parse("2023-09-18T14:49:01Z"));
+        store.fixUpdateTime(Instant.parse("2023-09-18T14:48:03Z"));
+        store.jobStarted(compactionJobStarted(startedJob, Instant.parse("2023-09-18T14:48:00Z")).taskId("started-task").build());
+        store.jobStarted(compactionJobStarted(uncommittedJob, Instant.parse("2023-09-18T14:48:01Z")).taskId("finished-task").build());
+        store.jobStarted(compactionJobStarted(finishedJob, Instant.parse("2023-09-18T14:48:02Z")).taskId("finished-task").build());
```

Comment:
This has the uncommitted and finished jobs both running on the same task at the same time. It's a nitpick but that's not possible. Can we put the uncommitted one on a separate task?